tests/Unit/ApparentHorizons/Test_SpherePackIterator.cpp

```diff
@@ -0,0 +1,46 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include <catch.hpp>
+
+#include "ApparentHorizons/SpherePackIterator.hpp"
+#include "tests/Unit/TestHelpers.hpp"
+
+SPECTRE_TEST_CASE("Unit.ApparentHorizons.SpherePackIterator",
+                  "[ApparentHorizons][Unit]") {
+
+  /// [spherepack_iterator_example]
+  size_t l_max = 4;
+  size_t m_max = 3;
+  SpherePackIterator iter(l_max,m_max); // stride is 1.
```

Comment:
Please clang-format this file -- this line and a few others have non-conforming whitespace.